frontend/src/cljs/imas_seamap/state_of_knowledge/utils.cljs

```diff
@@ -0,0 +1,17 @@
+;;; Seamap: view and interact with Australian coastal habitat data
+;;; Copyright (c) 2017, Institute of Marine & Antarctic Studies.  Written by Condense Pty Ltd.
+;;; Released under the Affero General Public Licence (AGPL) v3.  See LICENSE file for details.
+(ns imas-seamap.state-of-knowledge.utils
+  (:require #_[debux.cs.core :refer [dbg] :include-macros true]))
+
+;; Hardcoding this seems bad, but I'm not sure what to do about it? - TODO: figure out how to NOT hardcode this
```

Comment:
Yeah interesting question.  I'm not super fussed right now (we may revisit), but the answer would probably be in the layer categories (we were looking at those in the meeting earlier, funnily enough).  We already use those for a few things like "which layer is the transect-bathymetry layer", so could do something similar here.

More of a problem is that each boundary has its own specific handling, so I suspect that this bit of hard-coding is the least of your issues!  (Not sure there's any way around that, although maybe a few generic methods might encapsulate it a bit)